addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14448,13 +14458,13 @@ msgstr ""
 #. Description of settings category "Videos -> File lists" with label #14081
 #: system/settings/settings.xml
 msgctxt "#36176"
-msgid "Category containing settings for how video file lists are handled."
+msgid "This category contains the settings for how video file lists are handled."
 msgstr ""
 
 #. Description of setting "Videos -> File lists -> Default select action" with label #22079
 #: system/settings/settings.xml
 msgctxt "#36177"
-msgid "Toggle between Choose, Play (default), Resume and Show Information. Choose will select an item, e.g. open a directory in files mode. Resume will automatically resume videos from the last position that you were viewing them, even after restarting the system."
+msgid "Toggle between \"Choose\", \"Play\" (default), \"Resume\" and \"Show information\". \"Choose\" will select an item (e.g. open a directory in files mode.) \"Resume\" will automatically resume videos from the last position that you were viewing them, even after restarting the system."
```

Comment:
Agreed that old description looks to be wrong, suggested change looks good to me.